traffic_ops/testing/api_contract/v4/requirements.txt

```diff
@@ -0,0 +1,8 @@
+Apache-TrafficControl==3.1.0
+attrs==22.2.0
+munch==2.5.0
```

Comment:
what is `munch` used for? I don't see it being imported anywhere

Reply:
Removed it.